DigitalLearningSolutions.Web.Tests/ViewModels/LearningMenu/PostLearningAssessmentViewModelTests.cs

```diff
@@ -249,6 +250,210 @@ public void Post_learning_assessment_can_have_no_next_section()
 
             // Then
             postLearningAssessmentViewModel.NextSectionId.Should().BeNull();
+        }
+
+        [Test]
+        public void Post_learning_assessment_should_not_be_only_item_in_only_section_when_other_sections_exist()
+        {
+            // Given
+            var postLearningAssessment = PostLearningAssessmentHelper.CreateDefaultPostLearningAssessment(
+                otherSectionsExist: true,
+                otherItemsInSectionExist: false
+            );
+
+            // When
+            var postLearningAssessmentViewModel =
+                new PostLearningAssessmentViewModel(postLearningAssessment, CustomisationId, SectionId);
+
+            // Then
+            postLearningAssessmentViewModel.OnlyItemInOnlySection.Should().BeFalse();
+        }
+
+        [Test]
+        public void Post_learning_assessment_should_not_be_only_item_in_only_section_when_other_items_in_section_exist()
+        {
+            // Given
+            var postLearningAssessment = PostLearningAssessmentHelper.CreateDefaultPostLearningAssessment(
+                otherSectionsExist: false,
+                otherItemsInSectionExist: true
+            );
+
+            // When
+            var postLearningAssessmentViewModel =
+                new PostLearningAssessmentViewModel(postLearningAssessment, CustomisationId, SectionId);
+
+            // Then
+            postLearningAssessmentViewModel.OnlyItemInOnlySection.Should().BeFalse();
+        }
+
+        [Test]
+        public void Post_learning_assessment_should_have_be_only_item_in_only_section()
+        {
+            // Given
+            var postLearningAssessment = PostLearningAssessmentHelper.CreateDefaultPostLearningAssessment(
+                otherSectionsExist: false,
+                otherItemsInSectionExist: false
+            );
+
+            // When
+            var postLearningAssessmentViewModel =
+                new PostLearningAssessmentViewModel(postLearningAssessment, CustomisationId, SectionId);
+
+            // Then
+            postLearningAssessmentViewModel.OnlyItemInOnlySection.Should().BeTrue();
+        }
+
+        [Test]
+        public void Post_learning_assessment_should_be_only_item_in_this_section_when_no_other_sections_exist()
```

Comment:
Where do you specify that no other sections exist? And same for the test below? I'm not sure the test name reflect what you're testing. Maybe a parameterised test that checks expected OnlyItemInThisSection for all possible inputs of otherItemsInSectionExist and otherSectionsExist? And same for OnlyItemInOnlySection? What do you think? 🙂

Reply:
This is specified in line 311: `otherItemsInSectionExist: false`, essentially it checks that the value is passed through from the data model to the view model correctly. I'll parameterise these now 🙂

Reply:
But that's specifying that no other items exist in this section right? Whereas the test name says that no other sections exist 🙂

Reply:
Done in 407303e 🙂

Reply:
> But that's specifying that no other items exist in this section right? Whereas the test name says that no other sections exist 🙂

Yeah that test name was wrong, but it was removed when I parameterised the tests 🙂